src/s_path.c

```diff
@@ -259,7 +259,7 @@ void sys_setextrapath(const char *p)
     sys_expandpath("%CommonProgramFiles%/Pd", pathbuf, MAXPDSTRING);
     STUFF->st_staticpath = namelist_append(STUFF->st_staticpath, pathbuf, 0);
 #endif
-    /* add built-in "extra" path last so its checked last */
+*/
     STUFF->st_staticpath = namelist_append(STUFF->st_staticpath, p, 0);
```

Comment:
if you want to get rid of code, remove it, rather than commenting it out (and having to remove useful (explanatory) comments because C-comments don't stack).